pkg/interface/src/views/landscape/components/Graph/GraphContent.tsx

```diff
@@ -34,6 +34,72 @@ interface GraphMentionNode {
   ship: string;
 }
 
+const addEmphasisToMention = (contents: Content[], content: Content, index: number) => {
+  const prevContent = contents[index - 1];
```

Comment:
Will this ever go out of range?

Reply:
prevContent/nextContent being undefined is handled in the conditions on 63, 65, 75, and 79.